content/csv_download.md

```diff
@@ -0,0 +1,27 @@
+---
+title: CSV Download
+---
+
+## [Download here]({{ site.baseurl }}/assets/data/datasets.csv)
+
+This `.csv` file was created by parsing all currently existing dataset entries.
+It does not contain any new information, but is intended to facilitate deriving statistics or similar analysis efforts.
```

Comment:
First part is confusing IMHO.
Maybe "It can be used to sort and filter data in a spreadsheet program or generate statistics and plots."